features/upgrade/etcd/upgrade.feature

```diff
@@ -29,4 +29,31 @@ Feature: basic verification for upgrade testing
     When I use the "openshift-operators" project
     Then status becomes :running of exactly 1 pods labeled:
       | name=etcd-operator-alm-owned |
-
+	    
+  # @author knarra@redhat.com
+  # @case_id OCP-22665
+  @upgrade-check
+  @admin
+  Scenario: Check etcd image have been udpated to target release value after upgrade
+    # operands
+    When I run the :get admin command with:
+      | resource | po                                            |
+      | n        | openshift-etcd                                |
+      | o        | jsonpath={.items[0].spec.containers[0].image} |
+    Then the step should succeed
+    And evaluation of `@result[:response]` is stored in the :etcd_image clipboard
+    When I run the :get admin command with:
```

Comment:
@kasturinarra same applies to 45-49.  You can replace it with 
```
And evaluation of `cluster_version('version').image` is stored in the :payload_image clipboard
```

Reply:
addressed !!